AppointmentRazor/Pages/_ViewImports.cshtml

```diff
@@ -3,6 +3,7 @@
 @using Utilities.Localization
 @using Utilities.Authentication
 @using ViewModels
+@using AppointmentModel.Model
```

Comment:
Do we want to use the model in every page? If not maybe we should move the `@using` statement to be only in the pages that use it?